### ISSUE TITLE
No longer able to see my virtual machine on my mac m1

### ISSUE THREAD

**Issue by dk00127256** [utmapp/UTM#2736]
I had followed the steps as per the guide and everything was working.I am new to UMT and was a big fan of it until now where after reboot my windows virtual box is not seen I checked every other way rebooted my mac 2 times same problem

Did anyone face this before ? seems to be unreliable product

Regards
DJ

**Comment by conath** [comment]
Thanks for your message. I'm afraid I don't follow. You said, 
> windows virtual box is not seen

Did the VM disappear from the list in UTM? This would mean that the Folder created by UTM for this VM has been moved or deleted. 

You can check if the folder is there by creating and saving a new VM. Then right click it in the list (two finger tap on Trackpad) and choose Show in Finder. The Finder will open with a window showing the VM folders. Is the Windows VM there?

Note that UTM will never delete a VM without a confirmation message.

**Comment by dk00127256** [comment]
Hi

I see the windows10_.... ARM64_en-us_21354.qcow2 file with 9.71 GB 

Also I cannot create new VM after I name it and save nothing works.And if I try to import my VM file it does not allow to do that?

I did not select anything which will delete the files also just shutdown the VM then rebooted the machine and after that everything is gone 

Regards
DJ

**Comment by conath** [comment]
Do you have storage space available on your Mac?

**Comment by dk00127256** [comment]
Yes 140 GB available.This seems to be some issue definitely with UMT and Mac M1.Can you send me some updated version for UMT

**Comment by conath** [comment]
This has not been reported before, and we have not discovered any information on what causes the issue. So therefore I can't provide you with a solution yet. I would recommend you move the qcow2 file away from the folder where it is now, then delete UTM and re-download it. Other than that please tell me your macOS version and UTM version as well as how you installed it.

**Comment by dk00127256** [comment]
UMT : 2.1.2
macOS Big Sur 11.5.2

**Comment by conath** [comment]
Thanks. The versions are good, I have similar setup (and I don’t have this issue). You wrote 

> Also I cannot create new VM after I name it and save nothing works.And if I try to import my VM file it does not allow to do that?

Do you see an error message when you try this? If not, please open Console.app. Search for UTM in the top right, then click clear. Then open UTM and try to save a new VM. After that go back to Console app and [export all the messages](https://apple.stackexchange.com/questions/376196/how-to-export-the-event-log-from-the-console-app-into-a-text-file#376222), and upload here. This will allow me to check for a hidden error. Thanks.

I would do this myself if I could reproduce the error, but I can’t.

**Comment by GiacomoPaltrinieri** [comment]
i just had the same problem after closing UTM and opening it again.
Has any fix been released or is it just?

**Comment by PowerRangers1993** [comment]
I lost my VM also it is the blue one with the white lines I have MacBook Air 2024 model I cant find it what do I do?
